common/common_test.go

```diff
@@ -20,24 +20,28 @@ import (
 	"path/filepath"
 	"testing"
 
-	cmp "github.com/google/go-cmp/cmp"
 	slsa "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v0.2"
 	"github.com/project-oak/transparent-release/internal/testutil"
 	"github.com/project-oak/transparent-release/pkg/amber"
 )
 
-const testdataPath = "../testdata/"
-const provenanceExamplePath = "schema/amber-slsa-buildtype/v1/example.json"
+const (
+	testdataPath             = "../testdata/"
+	provenanceExamplePath    = "schema/amber-slsa-buildtype/v1/example.json"
+	wantTomlHash             = "f8b8b42df90a32f24822a5cf527574598a7812209455cf293bf99b84de90934b"
```

Comment:
I think most initialisms are incorrectly capitalised in this PR. See http://go/go-style/decisions#initialisms